src/scion/math/PolynomialSeries/src/evaluate.hpp

```diff
@@ -0,0 +1,18 @@
+/**
+ *  @brief Evaluate the function for a value of x
+ *
+ *  @param x   the value to be evaluated
+ */
+Y evaluate( const X& x ) const {
+
+  if ( ! this->isInside( x ) ) {
+
+    Log::error( "Legendre polynomials are defined for x values in the [-1,1] interval" );
```

Comment:
Hey missed that this says legendre polynomials instead of just polynomials. Maybe revise this error message.

Reply:
I saw this one later on, it is fixed in #8 (there was also a typo in the verifyCoefficients function)